src/libstore/build/local-derivation-goal.cc

```diff
@@ -2926,16 +2926,22 @@ bool LocalDerivationGoal::isReadDesc(int fd)
 
 StorePath LocalDerivationGoal::makeFallbackPath(OutputNameView outputName)
 {
+    // This is a bogus path type, constructed this way to ensure that it doesn't collide with any other store path
+    // FIXME: We may want to separate the responsibilities of constructing the path fingerprint and of actually doing the hashing
```

Comment:
```suggestion
    // TODO: Consider separating the responsibilities of constructing the path fingerprint and of actually doing the hashing
```

It's not broken.

Reply:
There's nothing to be _fixed_ here; just a refactor.

```suggestion
    // TODO: We may want to separate the responsibilities of constructing the path fingerprint and of actually doing the hashing
```